python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -784,6 +784,27 @@ def get_edge_data(self, edge_ids=None, types=None, columns=None):
 
         return None
 
+    def fillna(self, val=0, props_to_fill="both"):
```

Comment:
Is filling both edges and vertices a common use case? This would be our first API that let's us choose between vertices, edges, or both (it's not a bad pattern, but it is new).

I would probably create two methods, `fillna_vertices` and `fillna_edges`. I would also add e.g. `columns=None` optional argument to specify a subset of columns to fill.

Reply:
that sounds reasonable, I will go ahead and make this change